src/main.rs

```diff
@@ -672,26 +673,21 @@ fn cmd_certify(out: &mut dyn Write, cfg: &Config, sub_args: &CertifyArgs) -> Res
     }
 
     // Print out the EULA and prompt
-    write!(
-        out,
-        "I, {}, certify that I have audited ",
-        user_info.username
-    )?;
-    match &kind {
+    let what_version = match &kind {
         AuditKind::Full { version, .. } => {
-            write!(out, "version {} ", version)?;
+            format!("version {}", version)
         }
         AuditKind::Delta { delta, .. } => {
-            write!(out, "from version {} to {} ", delta.from, delta.to)?;
+            format!("the changes from version {} to {}", delta.from, delta.to)
         }
         AuditKind::Violation { .. } => unreachable!(),
-    }
-    writeln!(
-        out,
-        "of {} in accordance with the following criteria:",
-        sub_args.package
-    )?;
-    writeln!(out, "{}\n", eula)?;
+    };
+    let statement = format!(
+        "I, {}, certify that I have audited {} of {} in accordance with the following criteria:",
+        user_info.username, what_version, sub_args.package,
+    );
+    write!(out, "\n{}\n\n", textwrap::fill(&statement, 80).yellow().bold())?;
```

Comment:
So part of the reason I imported the Term library just for one little stdin below is because it also has stuff for coloured output: https://github.com/mitsuhiko/insta/blob/master/cargo-insta/src/cli.rs#L186-L202

I just haven't had the energy to dig into the details and also want to look into what the story is for properly enabling/disabling styling based on what you're outputting to. Not sure if my `write!(out)` machinery needs to be completely replaced to handle that or what :s

Reply:
Switched over to that.